examples/energy-management-app/esp32/main/Kconfig.projbuild

```diff
@@ -185,3 +185,38 @@ depends on ENABLE_PW_RPC
             about available pin numbers for UART.
 
 endmenu
+
+
+menu "Energy Management Example App Configuration"
+
+    config ENABLE_EXAMPLE_EVSE_DEVICE
+        bool "Enable EVSE device on EP1"
+        default "y"
+        help
+            To use Energy Management App as EVSE device and compile dependent files.
+
+    config ENABLE_EXAMPLE_WATER_HEATER_DEVICE
+        bool "Enable Water Heater device on EP2"
```

Comment:
Is this on EP2 or still EP1?

Reply:
Still on EP1. I'll change the comment to EP1